azure-devops/azext_devops/dev/boards/arguments.py

```diff
@@ -20,3 +20,9 @@ def load_work_arguments(self, _):
     with self.argument_context('boards work-item delete') as context:
         context.argument('yes', options_list=['--yes', '-y'], action='store_true',
                          help='Do not prompt for confirmation.')
+
+    with self.argument_context('boards work-item relation') as context:
+        context.argument('id', help='The ID of the work item')
+        context.argument('relation_type', help='Relation type to create. example: parent, child ')
```

Comment:
You will have to split the help text this shows up as create for both add and remove.

Reply:
nice catch